CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-14
+
+> TRUE or FALSE, it's a problem!
 
+### Important Notes
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
```

Comment:
Wonder how this did not made its way into rc1 release notes...

Reply:
We released rc1 on April 03, but that PR was merged on Apr 26.

Reply:
Or do you mean remove this `Important Notes` section? If yes, I think this two notes are so important that we should tell to the users. So I add this section.

Reply:
I asked about #3452, but now I see that #4248 (which comes with this changelog entry) was merged after rc.1. So it's fine.